modules/agent_files/files/home/jenkins-agent/cleanup_docker_images.py

```diff
@@ -61,7 +61,7 @@ def run_image_cleanup(args, minimum_age, dclient):
     processed_images = set()
     for i in reversed(images):
         dockerid = i.id
-        repo_tags = i.attrs['RepoTags'][0] if '<none>:<none>' not in i.attrs['RepoTags'] else dockerid
+        repo_tags = i.attrs['RepoTags'][0] if i.attrs['RepoTags'] else dockerid
```

Comment:
Should we access `i.tags` directly then? My understanding was that `RepoTags` had the `<none:none>` but the tags attribute had it stripped out, that's why I left it to keep the changes minimal.